THIRDPARTYLICENSES

```diff
@@ -289,6 +293,11 @@ DOMPurify - XSS sanitizer for HTML, MathML and SVG
 Copyright (C) 2024 by Dr.-Ing. Mario Heiderich, Cure53
 MPL-2.0 or Apache-2.0
 
+l10n.js
+By Eli Grey, http://eligrey.com
```

Comment:
Nit: I assume this line is not really needed, elsewhere we only have name-copyright-license lines.